### PR TITLE
Enable the upgrade to WoltLab Suite 5.5

### PR DESCRIPTION
**Pull request by dtdesign**
Based on 41f100782ce6abe92f144810b719c15e53bf4849

### CODE REVIEW

File: wcfsetup/install/lang/en.xml [hunk 1]
```diff
@@ -2006,9 +2006,11 @@ If you have <strong>already bought the licenses for the listed apps</strong>, th
...
 		<item name="wcf.acp.package.enableUpgradeOverride"><![CDATA[Upgrade to WoltLab Suite {$version}]]></item>
-		<item name="wcf.acp.package.enableUpgradeOverride.enable"><![CDATA[Enable Upgrading to the New Version]]></item>
+		<item name="wcf.acp.package.enableUpgradeOverride.enable"><![CDATA[Enable Upgrading to the Evaluation Version]]></item>
 		<item name="wcf.acp.package.enableUpgradeOverride.issues"><![CDATA[The following issues currently prevent an upgrade:]]></item>
-		<item name="wcf.acp.package.availableUpgradeVersion"><![CDATA[New Version Available!<br><br>The new WoltLab Suite {$version} is available for installation. We have provided a guide with the instructions for the upgrade in our <a href="https://manual.woltlab.com/en/updates/#upgrading-woltlab-suite-52-to-53" class="externalURL">manual</a>.<br>
+		<item name="wcf.acp.package.availableUpgradeVersion"><![CDATA[The new WoltLab Suite {$version} is available for installation. We have provided a guide with the instructions for the upgrade in our <a href="https://manual.woltlab.com/en/updates/#upgrading-woltlab-suite-52-to-53" class="externalURL">manual</a>.<br>
```
Comment (TimWolla) [inline wcfsetup/install/lang/en.xml:2011]: Ditto. Here the URL even is incorrect for 5.3 → 5.4.

File: wcfsetup/install/lang/de.xml [hunk 2]
```diff
@@ -2021,9 +2021,11 @@ Die Datenbestände werden sorgfältig gepflegt, aber es ist nicht ausgeschlossen
...
 		<item name="wcf.acp.package.enableUpgradeOverride"><![CDATA[Aktualisierung auf WoltLab Suite {$version}]]></item>
-		<item name="wcf.acp.package.enableUpgradeOverride.enable"><![CDATA[Upgrade auf die neue Version freigeben]]></item>
+		<item name="wcf.acp.package.enableUpgradeOverride.enable"><![CDATA[Upgrade auf die Testversion freigeben]]></item>
 		<item name="wcf.acp.package.enableUpgradeOverride.issues"><![CDATA[Die folgenden Probleme verhindern aktuell eine Aktualisierung:]]></item>
-		<item name="wcf.acp.package.availableUpgradeVersion"><![CDATA[Neue Version verfügbar!<br><br>Die neue WoltLab Suite {$version} steht zur Installation bereit. Eine Anleitung zur Durchführung des Upgrades steht im <a href="https://manual.woltlab.com/de/updates/#update-von-woltlab-suite-53-auf-54" class="externalURL">Handbuch</a> zur Verfügung.<br>
+		<item name="wcf.acp.package.availableUpgradeVersion"><![CDATA[Die neue WoltLab Suite {$version} steht zur Installation bereit. Eine Anleitung zur Durchführung des Upgrades steht im <a href="https://manual.woltlab.com/de/updates/#update-von-woltlab-suite-53-auf-54" class="externalURL">Handbuch</a> zur Verfügung.<br>
```
Comment (TimWolla) [inline wcfsetup/install/lang/de.xml:2026]: The URL in the manual needs to be updated.

**Review by TimWolla — COMMENTED** [review]
Other than that this LGTM.

**Review by TimWolla — APPROVED** [review]

File: wcfsetup/install/lang/de.xml [hunk 3]
```diff
@@ -2021,9 +2021,11 @@ Die Datenbestände werden sorgfältig gepflegt, aber es ist nicht ausgeschlossen
...
-		<item name="wcf.acp.package.enableUpgradeOverride.enable"><![CDATA[Upgrade auf die neue Version freigeben]]></item>
+		<item name="wcf.acp.package.enableUpgradeOverride.enable"><![CDATA[Upgrade auf die Testversion freigeben]]></item>
 		<item name="wcf.acp.package.enableUpgradeOverride.issues"><![CDATA[Die folgenden Probleme verhindern aktuell eine Aktualisierung:]]></item>
-		<item name="wcf.acp.package.availableUpgradeVersion"><![CDATA[Neue Version verfügbar!<br><br>Die neue WoltLab Suite {$version} steht zur Installation bereit. Eine Anleitung zur Durchführung des Upgrades steht im <a href="https://manual.woltlab.com/de/updates/#update-von-woltlab-suite-53-auf-54" class="externalURL">Handbuch</a> zur Verfügung.<br>
+		<item name="wcf.acp.package.availableUpgradeVersion"><![CDATA[Die neue WoltLab Suite {$version} steht zur Installation bereit. Eine Anleitung zur Durchführung des Upgrades steht im <a href="https://manual.woltlab.com/de/updates/#update-von-woltlab-suite-54-auf-55" class="externalURL">Handbuch</a> zur Verfügung.<br>
+<br>
+ACHTUNG: Es handelt sich aktuell noch um eine Testversion die nicht für den produktiven Einsatz geeignet ist!<br>
```
Comment (cadeyrn) [inline wcfsetup/install/lang/de.xml:2028]: missing comma: „[…] um eine Testversion, die nicht für den produktiven Einsatz […]“

**Pull request merged**